modules/core/src/main/java/org/apache/ignite/mxbean/IgniteMXBean.java

```diff
@@ -27,36 +27,152 @@
  */
 @MXBeanDescription("MBean that provides access to Kernal information.")
 public interface IgniteMXBean {
+    /** */
+    public String ATTR_DESC_FULL_VER = "String presentation of the Ignite version.";
```

Comment:
Can we rename this constant to `FULL_VER_DESC`?
Other constants introduced by this change should be renamed with this pattern, also.

Reply:
done